docs/modules/exploratory/fpca.rst

```diff
@@ -0,0 +1,30 @@
+Functional Principal Component Analysis (FPCA)
+==============================================
+
+This module provides tools to analyse functional data using FPCA. FPCA is
+a common tool used to reduce dimensionality while preserving the maximum
+quantity of variance in the data. FPCA be applied to a functional data object
+in either a basis representation or a discretized representation. The output
+of FPCA are orthogonal functions (usually a much smaller sample than the input
+data sample) that represent the most important modes of variation in the
+original data sample.
+
+For a detailed example please view `FPCA example
+<../../auto_examples/plot_fpca.html>`_, where the process is applied to several
```

Comment:
I recall there was a better way to link to examples

Reply:
Yes, I am changing to that way right now.

Reply:
I used this one because it is the first one I found, but after viewing other examples I found the one you mentioned that does not depend on the directory we are.